### DISCUSSION TITLE
en/stable/examples/workflows/upstream/workflow_template__dag/

### DISCUSSION THREAD

**Discussion by giscus** [argoproj-labs/hera#1090] [Website Discussions]
# en/stable/examples/workflows/upstream/workflow_template__dag/

https://hera.readthedocs.io/en/stable/examples/workflows/upstream/workflow_template__dag/

**Comment by jiantang99** [comment]
Hi, I am trying to use this method but I am running into this error:
hera.exceptions.InternalServerError: Server returned status code 500 with message: `templates.main.tasks.<template name> template reference <template name>.<template ref> not found`
Do you have any idea what's causing this? 😥

**Reply by flaviuvadan** [reply]
Does the template exist in your Argo Workflows instance? From `not found` I guess that answer _may_ be no but thought of checking

**Reply by jiantang99** [reply]
The template exists on the argo workflow server, is it possible to reference this template using this method?
